service/history/timerQueueProcessor.go

```diff
@@ -143,7 +143,9 @@ func (t *timerQueueProcessorImpl) FailoverDomain(domainID string) {
 	failoverTimerProcessor := newTimerQueueFailoverProcessor(t.shard, t.historyService, domainID,
 		standbyClusterName, minLevel, maxLevel, t.matchingClient, t.logger)
 	failoverTimerProcessor.Start()
-	failoverTimerProcessor.timerQueueProcessorBase.readAndFanoutTimerTasks()
+	// use a fake timer to trigger db scan
```

Comment:
'readAndFanoutTimerTasks' is still called from other places and I thought that is the one which could block.

Reply:
readAndFanoutTimerTasks will be called by an separate go routine, so we should be fine.